pilot/prompts/components/steps_list.prompt

```diff
@@ -0,0 +1,24 @@
+{% if task_steps and step_index -%}
```

Comment:
Here are some examples of prompts with these changes:

<img width="1213" alt="Screenshot 2024-01-28 at 8 49 17 PM" src="https://github.com/Pythagora-io/gpt-pilot/assets/41999013/c6f2f88c-3716-488d-a962-745c891b2375">
<img width="966" alt="Screenshot 2024-01-28 at 8 59 22 PM" src="https://github.com/Pythagora-io/gpt-pilot/assets/41999013/30d7759d-6067-4b27-9bb2-7fac99706abc">
<img width="1269" alt="Screenshot 2024-01-28 at 9 44 19 PM" src="https://github.com/Pythagora-io/gpt-pilot/assets/41999013/0ebc079b-9d69-4555-a89a-763da3d3a856">